src/renderer/redux/actions/content.js

```diff
@@ -139,7 +140,14 @@ export function doUpdateLoadStatus(uri, outpoint) {
             0
           );
 
-          if (selectosNotificationsEnabled(getState())) {
+          if (state.navigation.currentPath === '/subscriptions') {
+            dispatch(
+              doNotify({
+                message: `'${fileInfo.metadata.title}' has been downloaded`,
+                displayType: ['snackbar'],
```

Comment:
I think the idea behind snackbars is that they are reserved for user actions, which wouldn't make sense for this use case. Maybe we should create our own notification component that mimics desktop notifications but has some additional style. 

Not needed for this PR but I think we should stick with the OS notifications for now.